infra/backup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# SSH into our Clerk EC2 instance
+# Backup the Postgres DB and upload to S3
+#
+set -e
+HOST='13.55.250.149'
+TIME=$(date "+%s")
+DATABASE_NAME=clerk
+S3_BUCKET=s3://anika-database-backups
+BACKUP_FILE="postgres_${DATABASE_NAME}_${TIME}.sql.gz"
+S3_PATH="$S3_BUCKET/$BACKUP_FILE"
+
+if [[ -z "$CLERK_PRIVATE_SSH_KEY" ]]
+then
+    echo -e "\n>>> Error: Clerk private key not found in CLERK_PRIVATE_SSH_KEY."
+    exit 1
+fi
+
+echo -e "\n>>> Backing up Postgres DB on Clerk EC2 instance at $HOST."
+
+echo -e "\n>>> Setting up private key."
+echo -e "$CLERK_PRIVATE_SSH_KEY" > private.key
+chmod 600 private.key
+
+echo -e "\n>>> SSHing into Clerk EC2 instance at $HOST."
+ssh -o StrictHostKeyChecking=no -i private.key root@$HOST /bin/bash << EOF
+    set -e
+    cd /srv/backups
+    . env/bin/activate
+    touch clerk.log
```

Comment:
both the virtualenv and the logfile aren't required anymore

- virtualenv: `pg_dump` and `aws` are both already installed on the clerk box so there's no need for this
- logfile: github actions will store action logs for us (for a while at least)

please remove the use of virtualenv and logfiles

Reply:
@MattSegal 
- we currently SSH into our Clerk instance and then write detailed information to `clerk.log`
- how does GA store logs for us? I assume that any info from operations we perform whilst we have an SSH connection to Clerk instance will not be logged by GA. 
- I assume that GA logs info from commands such as `echo -e "Have done such and such"` - in that case we might need to insert additional echo statements?

Reply:
hey Peter all stdout, including that from a SSH session should be captured by githhub actions and be accessible via the github UI